### PR TITLE
[12.0] [FIX] product_pricelist_direct_print: Fixed translations on xlsx generator

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT27988

This is the fix to make the titles of the xlsx translatables.

Please @pedrobaeza @carlosdauden review this

### CODE REVIEW

File: product_pricelist_direct_print/i18n/es.po [hunk 1]
```diff
@@ -294,6 +335,12 @@ msgstr "Plantillas de producto"
 msgid "Products"
 msgstr "Productos"
 
+#. module: product_pricelist_direct_print
+#: code:addons/product_pricelist_direct_print/report/product_pricelist_xlsx.py:65
+#, python-format
+msgid "Sale Price"
+msgstr "Precio ficha"
```
Comment (pedrobaeza) [inline product_pricelist_direct_print/i18n/es.po]:
```suggestion
msgstr "PVP"
```

File: product_pricelist_direct_print/report/product_pricelist_xlsx.py [hunk 2]
```diff
@@ -34,18 +34,18 @@ def _create_product_pricelist_sheet(self, workbook, book, pricelist):
...
         sheet.set_column("B:H", 15)
         # Title construction
-        sheet.write('A1', 'Price List Name:', title_format)
+        sheet.write('A1', _('Price List Name:'), title_format)
         if not book.hide_pricelist_name:
             sheet.write('A2', pricelist.name)
         else:
-            sheet.write('A2', "Special Pricelist")
-        sheet.write('B1', 'Currency:', title_format)
+            sheet.write('A2', _("Special Pricelist"))
```
Comment (carlosdauden) [inline product_pricelist_direct_print/report/product_pricelist_xlsx.py]: Use same character in all strings (" or ')

File: product_pricelist_direct_print/report/product_pricelist_xlsx.py [hunk 3]
```diff
@@ -99,7 +99,7 @@ def _fill_data(self, workbook, sheet, book, pricelist):
                 )
                 row += 1
         if book.summary:
-            sheet.write(row, 0, "Summary:", bold_format)
+            sheet.write(row, 0, _("Summary:"), bold_format)
```
Comment (carlosdauden) [inline product_pricelist_direct_print/report/product_pricelist_xlsx.py]: Same

**Review by carlosdauden — APPROVED** [review]
Attend @pedrobaeza comments

**Comment by CarlosRoca13** [comment]
@carlosdauden @pedrobaeza changes done

**Pull request merged**